Python/operators.py

```diff
@@ -0,0 +1,81 @@
+import warnings
+
+import xarray as xr
+import numpy as np
+
+def check_keys(data_cube):
+    rlat = 'rlat' in data_cube.variables
+    rlon = 'rlon' in data_cube.variables
+    run = 'run' in data_cube.variables
+    if not rlat and rlon and run:
+        raise KeyError("CanRCM4 ensemble is missing 'run' axis, 'rlon' \
+                        and 'rlat'. Please ensure the model is properly \
+                        gridded with these names.")
+
+def cell_count(data_cube):
+    check_keys(data_cube)
+    n = data_cube['run'].shape[0]
+    p = data_cube['rlat'].shape[0]*data_cube['rlon'].shape[0]
+    return n, p
+
+def center_data(data_cube):
+    """Centers each run in data_cube
+    by subtracting ensemble mean at
+    a given grid cell.
+    --------------------------------
+    Args: 
+        data_cube (xarray Dataset): datacube 
+            containing an ensemble of 
+            CanRCM4 models
+    Returns:
+        X (xarray Dataset): mean centered
```

Comment:
Perhaps there is a scientific reason for calling the variable `X` but I think it would make it easier to understand if you gave it a more descriptive name.